### PR TITLE
Added Queue status.

### PR DESCRIPTION
**Pull request by k82cn**
Signed-off-by: Da K. Ma <klaus1982.cn@gmail.com>

**Release note**:
```release-note
Added number of PodGroup in different state.
```

### CODE REVIEW

File: config/crds/scheduling_v1alpha1_podgroup.yaml [hunk 1]
```diff
@@ -41,3 +41,11 @@ spec:
           type: object
       type: object
   version: v1alpha1
+  subresources:
+    status: {}
+status:
```
Comment (hex108) [inline config/crds/scheduling_v1alpha1_podgroup.yaml]: We could omit it if all fields are empty?

Comment (k82cn) [reply config/crds/scheduling_v1alpha1_podgroup.yaml]: if omit, `UpdateStatus` will not work :)

Comment (hex108) [reply config/crds/scheduling_v1alpha1_podgroup.yaml]: Then it is OK. :) 

But in my CRD use case,
```
  subresources:
    status: {}
```
is enough.

Comment (k82cn) [reply config/crds/scheduling_v1alpha1_podgroup.yaml]: great to know, I'll remove the other part :)

Comment (k82cn) [reply config/crds/scheduling_v1alpha1_podgroup.yaml]: done

**Comment by hex108** [comment]
LGTM

**Pull request merged**